long_cat_is_long.py

```diff
@@ -0,0 +1,35 @@
+import string
+from collections.abc import Generator
+
+
+def words(text: str) -> Generator[str]:
```

Comment:
Function and varaible names should be expressive and not generic. It is unclear from reading the function names what it's purpose is.